src/components/BooksPage.js

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import { ListGroup, Button, Form } from 'react-bootstrap';
+import './BookPageStyle.css';
+
+const BooksPage = () => (
+  <main>
+    <section className="books-page">
+      <h1>Books List</h1>
+      <ListGroup>
+        <ListGroup.Item>
+          Cras justo odio
+          <Button variant="secondary">Remove</Button>
+        </ListGroup.Item>
```

Comment:
- [ ] Please create a separate component for the books, for example a `<Book />` component 😄